erpnext/erpnext_integrations/doctype/leaflink_settings/leaflink_settings.json

```diff
@@ -0,0 +1,108 @@
+{
+ "creation": "2021-04-12 17:03:28.589509",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "enable_leaflink",
+  "leaflink_id",
+  "api_key",
+  "api_value",
+  "incoming_api_authentication_key",
+  "orders_url",
+  "products_url",
+  "customers_url",
+  "column_break_7",
+  "default_customer_group",
+  "default_customer_territory"
+ ],
+ "fields": [
+  {
+   "depends_on": "eval: doc.enable_leaflink == 1",
```

Comment:
Add all of the fields that require `doc.enable_leaflink == 1` into a section and hide the whole section rather than marking individual fields in eval.